### ISSUE TITLE
HTTP ConnectionPool timeout

### ISSUE THREAD

**Issue by anjanaradhakrishnan** [Shuffle/Shuffle#1395]
Hello @frikky , I have integrated the shuffle with thehive for case creation. Whenever the hive is getting more than 8 requests at a time it is creating only 8 cases. For the remaining cases we are getting the below response.
<img width="347" alt="image" src="https://github.com/Shuffle/Shuffle/assets/105666732/636b26a8-394e-4f66-8c2b-32d34d7f71dd">
How to resolve this issue?

**Comment by frikky** [comment]
> Hello @frikky , I have integrated the shuffle with thehive for case creation. Whenever the hive is getting more than 8 requests at a time it is creating only 8 cases. For the remaining cases we are getting the below response. <img alt="image" width="347" src="https://private-user-images.githubusercontent.com/105666732/331979941-636b26a8-394e-4f66-8c2b-32d34d7f71dd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYyMDIwNzksIm5iZiI6MTcxNjIwMTc3OSwicGF0aCI6Ii8xMDU2NjY3MzIvMzMxOTc5OTQxLTYzNmIyNmE4LTM5NGUtNGY2Ni04YzJiLTMyZDM0ZDdmNzFkZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMFQxMDQyNTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMWFmODhkYzhjMzhkYTg3NjdhNzc5Yjk1ODRhZDIxM2RiMWJhZjJhYjY4Y2YwYzU2YjczMTVkOGNiZmFjN2JmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.51BGlm_IZnoBnh0qUAgjWhTryO_B0ayVUzTkhBeKzEM"> How to resolve this issue?

What do you mean 8 requests at a time? Does it work most of the time, but stop working when you send multiple in a row?

**Comment by anjanaradhakrishnan** [comment]
When creating cases in thehive via shuffle, at max 8 cases are created and rest of the cases results in HTTP pool connection timeout for a while.

**Comment by frikky** [comment]
> When creating cases in thehive via shuffle, at max 8 cases are created and rest of the cases results in HTTP pool connection timeout for a while.

That is a very weird arbitrary limit that I don't think has to do with Shuffle or TheHive directly. 

Can you send a sample input and the exact output you are getting? You can hide urls and such of course :)

**Comment by anjanaradhakrishnan** [comment]
Yeah so i tried creating cases in hive using the  below command in a loop for load testing which should create 10 cases. 
```curl -u xx@thehive.local:xx@123 http://xx.xx.xx.com/api/v1/case -H 'Content-Type: application/json' -d '{
  "title": "AWS Cloudtrail: signin.amazonaws.com - ConsoleLogin - User login failed.",
  "description": "AWS Cloudtrail: signin.amazonaws.com - ConsoleLogin - User login failed."
}'
```
But it is only creating 8 cases.  Rest of them are failing with the below error.
<img width="315" alt="image" src="https://github.com/Shuffle/Shuffle/assets/105666732/ad6e83c5-909f-4e4f-98da-c77fecd39d16">
